doc/rados/operations/monitoring.rst

```diff
@@ -366,12 +378,23 @@ on the number of replicas, clones and snapshots.
 
 - **NAME:** The name of the pool.
 - **ID:** The pool ID.
-- **USED:** The notional amount of data stored in kilobytes, unless the number 
-  appends **M** for megabytes or **G** for gigabytes.
+- **USED:** space allocated for a pool at all OSDs. This includes replication, allocation granularity and erasure coding overhead. Compression savings and object content gaps are taken into account as well. Internally reserved/allocated space (BlueFS part) isn't included.
```

Comment:
this should be rephrased to improve the English.

Also, rather than saying 'BlueFS part', which is a detail most users wouldn't know about, we could say 'BlueStore's db is excluded' like above.

Reply:
I've made this change. The English reads better to me now, but you'll have to vet the technical details.